test/controllers/greetings_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class GreetingsControllerTest < ActionDispatch::IntegrationTest
+  test "should get hello" do
+    get greetings_hello_url
```

Comment:
This test is currently failing. The `root_url` helper mentioned above might come in handy here.